dbfit-java/core/src/main/java/dbfit/fixture/FunctionResultSetStatementExecution.java

```diff
@@ -1,14 +1,12 @@
-package dbfit.environment;
+package dbfit.fixture;
 
 import java.sql.*;
 
-import dbfit.fixture.StatementExecution;
-
-public class InformixFunctionStatementExecution extends StatementExecution {
+public class FunctionResultSetStatementExecution extends StatementExecution {
```

Comment:
> Now contradicting the current PR title, I've extracted a DbStoredProcedureCall interface from the existing implementation. This does still include a reference to the StatementExecution implementation so maybe it would be good to create the others all in separate PR.
> 
> But, what do you think of this direction?

Looks like a potentially good approach to examine.